### PR TITLE
WinUI-3213 : Updated formatting value in UG document for SfMaskedTextBox control

### PR DESCRIPTION
**Pull request by Vignesh-SF3580**
### Description
Updated the formatting value page for SfMaskedTextBox control.

### Solution Description
Updated the contents for the formatting value in masked text box control.
Added the output images for the different types of mask formats.

### CODE REVIEW

**Comment by Vignesh-SF3580** [comment]
rebuild

File: winui/Masked-TextBox/Formatting-Value.md [hunk 1]
```diff
@@ -7,4 +7,135 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Value mask formats in WinUI SfMaskedTextBox
\ No newline at end of file
+# Value mask formats in WinUI SfMaskedTextBox
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Formatting-Value.md:10]: @Vignesh-SF3580 Formatting value in WinUI SfMaskedTextBox

File: winui/Masked-TextBox/Formatting-Value.md [hunk 2]
```diff
@@ -7,4 +7,135 @@ control: SfMaskedTextBox
...
+
+`SfMaskedTextBox` allows you to format the characters in the `Value` property in a mask scenario (when the Mask property is set). By default, the `Value` property holds your input characters, prompt characters and the literals defined in the mask. You can modify this and allow the `Value` property to hold the characters without prompt and literals by setting the `ValueMaskFormat` property of the control. The `Value` in the `SfMaskedTextBox` is formatted by any one of the following formatting enum values:
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Formatting-Value.md:12]: @Vignesh-SF3580 

The MaskedTextBox control allows you to format input values with prompt and literal characters which are defined in the mask expression by setting the ValueMaskFormat property. By default, the Value property of the control includes the characters typed by the user, as well as any prompt or literal characters in the input. The control provides several formatting options, which are listed below.

File: winui/Masked-TextBox/Formatting-Value.md [hunk 3]
```diff
@@ -7,4 +7,135 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Value mask formats in WinUI SfMaskedTextBox
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Formatting-Value.md:10]: @Vignesh-SF3580 Learn all about how to set the mask format for the value in the MaskedTextBox (SfMaskedTextBox) control with prompts and literals.

File: winui/Masked-TextBox/Formatting-Value.md (hunk 2 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Formatting-Value.md]: Changed the content for formatting value like this.

File: winui/Masked-TextBox/Formatting-Value.md (hunk 3 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Formatting-Value.md:10]: Changed the description.

File: winui/Masked-TextBox/Formatting-Value.md (hunk 1 above)
Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Formatting-Value.md]: Changed the content.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**